File: StudyImageProfile/StudyImageProfile/ViewController.m
//
//  ViewController.m
//  StudyImageProfile
//
//  Created by Casey on 17/01/2019.
//  Copyright © 2019 newplatform. All rights reserved.
//

#import "ViewController.h"
#import "TwoViewCtr.h"

@interface ViewController (){
    
    IBOutlet UIImageView *_imageView;
}

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    
    
}


- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    
    
    
    //_imageView.image = [UIImage imageNamed:@"11.jpg"];
    
    [self.navigationController pushViewController:[TwoViewCtr new] animated:YES];
    
    
}

@end

File: Refresh/Refresh/ViewController.swift
//
//  ViewController.swift
//  Refresh
//
//  Created by Casey on 05/04/2019.
//  Copyright © 2019 nb. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
    
    }

    
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        
        self.navigationController?.pushViewController(BunisessViewCtr(), animated: true)
        
    }

}

